src/actions/network/index.js

```diff
@@ -0,0 +1,24 @@
+import { networkSet as btcNetworkSet } from './btc';
+import { networkSet as lskNetworkSet } from './lsk';
+import actionTypes from '../../constants/actions';
+
+
+export const networkSet = data =>
+  (dispatch) => {
+    [
+      btcNetworkSet,
+      lskNetworkSet,
```

Comment:
we don't need to store a network object for any assets expect lisk. So, this action only set network config for lisk.

Reply:
I store the network object because we wanted to make things modular and generic. At the moment we don't need to store anything for BTC, but it is only because we don't allow BTC custom node. If we will want to have BTC custom node (e.g. for e2e tests) in the future, it will be easy to add.

I think it will be useful also for any other token in the future as we don't know what might be needed to store for other tokens.